script.js

```diff
@@ -81,11 +83,43 @@ function checkWin() {
   return false;
 }
 
+function checkDraw() {
+  if (
+    gameSpaces[0].innerHTML !== "" &&
+    gameSpaces[1].innerHTML !== "" &&
+    gameSpaces[2].innerHTML !== "" &&
+    gameSpaces[3].innerHTML !== "" &&
+    gameSpaces[4].innerHTML !== "" &&
+    gameSpaces[5].innerHTML !== "" &&
+    gameSpaces[6].innerHTML !== "" &&
+    gameSpaces[7].innerHTML !== "" &&
+    gameSpaces[8].innerHTML !== ""
+  ) {
+    return true;
+  }
+  return false;
+}
+
+function clearGameBoard(event) {
+  if (text.innerHTML === "PLAYER " + xGamePiece + " WINS!") {
+    text.innerHTML = "Player X turn";
+  }
+  if (text.innerHTML === "PLAYER " + oGamePiece + " WINS!") {
+    text.innerHTML = "Player O turn";
+  }
+  if (text.innerHTML === "DRAW!") {
+    text.innerHTML = "Player X turn";
+  }
+  gameSpaces.forEach((space) => (space.innerHTML = ""));
+}
+
 function useGamePiece(event) {
   playerTurn ? (gamePiece = xGamePiece) : (gamePiece = oGamePiece);
 
   event.target.innerHTML = gamePiece;
-  if (checkWin()) {
+  if (checkDraw()) {
```

Comment:
I think you may need to swap the order that the game checks for a draw and a win.
since the `checkDraw` function checks if every space is not an empty string than it will never know if there is a win. It will always be a draw since both a win and a draw mean all spaces are no longer empty strings.

So I think it makes more sense to check if there is a win and if there is not a win then check if there is a draw